docs/userguides/config.md

```diff
@@ -9,6 +9,15 @@ There are two locations you can place an `ape-config.yaml` file.
 Project settings take precedent, but global settings allow you to configure preferences across all projects, such as your default mainnet provider (e.g. Alchemy versus running your own node).
 
 This guide serves as an index of the settings you can include in any `ape-config.yaml` file.
+This guide is **PURPOSELY** alphabetized to facilitate easier look-up of keys.
+
+Most of the features in this guide are documented more-fully elsewhere in the user-guides.
+
+However, here is a list of common-use cases requiring the `ape-config.yaml` file to help you:
+
+1. Setting up a custom node RPC: See the `geth` section.
```

Comment:
Why a custom RPC provider is called `geth`? Most people do not use GoEthereum and this is very confusing for a reader.

Reply:
we are renaming it in 0.8, it is called geth because it starts a test local geth node when using local network, much like using foundry or hardhat, but it is also tightly coupled (currently) with generic node support, but we are changing that 0.8, thanks